src/mac/component/mod.rs

```diff
@@ -0,0 +1,10 @@
+cfg_if! {
```

Comment:
Missing header.

Reply:
All of the header and trailing newlines should be fixed 👍